web/src/utils/commons.ts

```
@@ -449,6 +449,36 @@ export const updatePanel = async (
+    store.dispatch("setAllDashboardListHash", {
+      [dashboardId]: hash,
+    });
+
+    store.dispatch("setDashboardData", {
+      [dashboardId]: convertedData,
+      hash: hash,
+    });
+
+    return store.state.organizationData.allDashboardData[dashboardId];
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)